### PR TITLE
Fogl 3473: tuning 4

### PR DESCRIPTION
**Pull request by sstefano69**

### CODE REVIEW

File: C/common/storage_client.cpp [hunk 1]
```diff
@@ -19,6 +20,14 @@
 #include <thread>
 #include <map>
 #include <string_utils.h>
+#include <sys/uio.h>
+#include <errno.h>
+
+#define INSTRUMENT	1
```
Comment (MarkRiddoch) [inline C/common/storage_client.cpp]: Let's set INSTRUMENT to 0 before checking in

**Review by MarkRiddoch — CHANGES_REQUESTED** [review]

File: C/common/storage_client.cpp (hunk 1 above)
Comment (sstefano69) [reply C/common/storage_client.cpp]: done

**Review by MarkRiddoch — APPROVED** [review]

**Comment by sstefano69** [comment]
obsolete

**Comment by sstefano69** [comment]
obsolete

**Pull request closed without merging**